### ISSUE TITLE
Review Accessibility of Website

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#407]
The website is updated at this URL: https://equalify.app

Please review the accessibility of the home page and individual post pages. Here are a few post pages:

- https://equalify.app/a11ys-ai-future-the-future-is-now/
- https://equalify.app/helping-wordpress-org-become-accessible/

**Comment by kevinandrews1** [comment]
@bbertucc:
- [x] Missing h1 on homepage. Maybe make "Efficient. Open. Accessible." the h1 since it's bolded (queried page for font type but screen readers don't differentiate font weight and size automatically)
- [x] "Verison" instead of Version in the alt text on homepage for decorative text image
- [ ] Duplicate/very similar link text on homepage. Confusing especially if pulling up a links list, a feature of modern screen readers (Book a Demo, Book a 15-Minute Demo, and multiple instances of Buy Now)

**Comment by bbertucc** [comment]
@kevinandrews1 - the first two are complete.

For the third point, I varied the text of the links. Does that work? We need to have those links multiple times on a page because as someone scrolls through content they should have a quick way to do what we want them to do (buy or book).

If it all looks good now, feel free to close this issue.
